src/components/comment/article.tsx

```diff
@@ -54,13 +54,6 @@ const topBorder = css`
     `}
 `;
 
-
-const commentLineStylePosition = css`
```

Comment:
Is this related to the footer PR?

Reply:
No this is from the "Lines" PR for positioning them behind the avatar